wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectServiceApi.java

```diff
@@ -680,7 +692,7 @@ public ProjectSearchResponseDto search(ProjectSearchRequestDto request) {
 
     try {
       return search(path, name, text, maxItems, skipCount);
-    } catch (ServerException | ConflictException | NotFoundException | ForbiddenException e) {
+    } catch (ServerException | NotFoundException | BadRequestException e) {
```

Comment:
Not sure it makes sense to throw these exceptions in other code to lose the information about exception type here.